radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -282,3 +284,27 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+/**
+ * Resets microcontroller of the antenna
+ *
+ *  @return 0 for success, non-zero for failure. See hal/errors.h for details.
+ */
+int antennaReset(void){
+
+	// Reset both side A/B antennas. See section 6.2 of Antenna System User Manual
+
+	int errorA = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY, isisants_sideA);  //reset on side A
+	int errorB = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY, isisants_sideB);  //reset on side B
+
+	if(errorA != 0) {
+			// TODO: record errors (if present) to System Manager
+			return errorA;
+		}
+	if(errorB != 0) {
+			// TODO: record errors (if present) to System Manager
+			return errorB;
+		}
```

Comment:
Instead of having 2 error variables and running both commands in sequence, I would recommend running a single reset command, checking the error (and returing it if there is one), and _then_ continuing on to the second command